--- src/make/Make.std-package-targets	2025-07-01 09:31:59.000000000 -0500
+++ src/make/Make.std-package-targets.bak	2025-08-28 16:17:34.992240839 -0500
@@ -58,12 +58,13 @@
         # set include path to those
         # lots of possible include paths to try to catch all cases of in situ, installed
         # and re-packages into UPS
-	  ROOT_INCLUDE_PATH=$(GENIE_LIB_PATH):$(GENIE)/src:$(GENIE)/src/$(PACKAGE) \
+	  ROOT_INCLUDE_PATH=$(GENIE_LIB_PATH):$(GENIE)/src:$(GENIE)/src/$(PACKAGE):$(TPYTHIA6_INC) \
           rootcling -f $(DICTIONARY).cc \
                     -I$(PACKAGE) -IGENIE \
                     -IGENIE/$(PACKAGE) \
                     -I../include -I../include/GENIE \
                     -I../include/GENIE/$(PACKAGE) \
+                    -I$(TPYTHIA6_INC) \
                     -inlineInputHeader \
                     -s $(PACKAGE_LIBNOVRS) \
                   -rml $(PACKAGE_PCM) \
